### DISCUSSION TITLE
where can find zh ```nemo_text_processing``` examples?

### DISCUSSION THREAD

**Discussion by jaffe-fly** [NVIDIA-NeMo/NeMo#2968] [Q&A] [answered]
in ```nemo_text_processing``` dir . i only find ```en```,```ru```  language ,where can i find ```zh```language examples ???

**Comment by yzhang123** [comment, marked as answer]
we don't have chinese support now since we lack the linguistic capabilities

**Reply by yzhang123** [reply]
would you like to do an open source contribution? we have a tutorial on how to add new languages? https://github.com/NVIDIA/NeMo/blob/main/tutorials/text_processing/WFST_Tutorial.ipynb
